src/msteams-app.css

```diff
@@ -0,0 +1,163 @@
+@charset "UTF-8";
+
+body {
+  font-family:  "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif !important;
+  font-size: 1.4rem;
+  min-width: auto;
+}
+a {
+  color: #5558af;
+}
+a:hover, a:focus {
+  text-decoration: underline;
+  outline: none;
+}
+
+/* Form controls */
+.form-field-title:first-of-type {
+  margin-top: 0;
+}
+.form-field-title {
+  display: flex;
+  margin: 1rem 0 .3rem;
+}
+.form-control:focus {
+  border-bottom-color: #5558af;
+  box-shadow: none;
+  outline: 0;
+}
+.form-control {

```

Comment:
This is intended for new code, our config experience, so there shouldn't be conflicting styles. I will try hierarchical styles for the default theme (which is what causes the problem because there's no class for it).